cli/api/catalog/catalog.go

```diff
@@ -23,6 +23,14 @@ import (
 	"fmt"
 	"github.com/apache/brooklyn-client/cli/models"
 	"github.com/apache/brooklyn-client/cli/net"
+	"net/url"
+        "path/filepath"
```

Comment:
indentation, worth adding `golint` and `go vet` to the build process to pick up stuff like this.